src/objects/zcl_abapgit_object_intf.clas.locals_imp.abap

```diff
@@ -457,25 +462,108 @@ CLASS lcl_aff_type_mapping IMPLEMENTATION.
     es_data = ls_data_aff.
   ENDMETHOD.
 
+  METHOD zif_abapgit_aff_type_mapping~to_abapgit.
+    DATA:
+      ls_data_abapgit TYPE zcl_abapgit_object_intf=>ty_intf,
+      ls_data_aff     TYPE zif_abapgit_aff_intf_v1=>ty_main,
+      lv_classname    TYPE seoclsname.
+
+
+    ls_data_aff = iv_data.
+
+    lv_classname = iv_object_name.
+
+    ls_data_abapgit-description = set_abapgit_descriptions( is_clsname  = lv_classname
+                                                            is_intf_aff = ls_data_aff ).
+
+    ls_data_abapgit-vseointerf-clsname = iv_object_name.
+    ls_data_abapgit-vseointerf-descript = ls_data_aff-header-description.
+    ls_data_abapgit-vseointerf-category = ls_data_aff-category.
+    ls_data_abapgit-vseointerf-unicode  = ls_data_aff-header-abap_language_version.
+    ls_data_abapgit-vseointerf-langu    = ls_data_aff-header-original_language.
+    ls_data_abapgit-vseointerf-clsproxy = ls_data_aff-proxy.
+    " to be reviewed
+    ls_data_abapgit-vseointerf-exposure = seoc_exposure_public.
+    ls_data_abapgit-vseointerf-state    = seoc_state_implemented.
+
+    "  to do : fill it right!
+*    ls_data_abapgit-vseointerf-author    = sy-uname.
+*    ls_data_abapgit-vseointerf-createdon = sy-datum.
+*    ls_data_abapgit-vseointerf-changedby = sy-uname.
```

Comment:
Is this still open? Or shall we remove these comments?

I think these values don't need to be set here. They should be set when the interface is created/updated in the system.

```suggestion
```

Reply:
fixed